kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue

```diff
@@ -87,13 +87,13 @@
         if (!this.pageReady) {
           return null;
         }
-        return this.pdfPage.view[3];
+        return this.pdfPage.view[3] - this.pdfPage.view[1];
```

Comment:
Could you add code comments here to add your explanation from the PR description into the code for the benefit of future readers?